include/openenclave/bits/sgx/sgxtypes.h

```diff
@@ -1117,14 +1117,14 @@ typedef enum _quote3_error_t {
 // clang-format on
 
 /*
- * Define enclave_elrange_t, used by enclave_create_ex.
+ * Define sgx_enclave_elrange_t, used by enclave_create_ex.
  */
-typedef struct enclave_elrange
+typedef struct sgx_enclave_elrange
 {
     uint64_t enclave_image_address;
     uint64_t elrange_start_address;
     uint64_t elrange_size;
-} enclave_elrange_t;
+} sgx_enclave_elrange_t;
```

Comment:
public oe types should start with `oe_`

Reply:
This is not a publicly visible structure, and used similar to sgx_sigstruct_t. This parameter is expected by the APIs that intel/linux-sgx makes available to us..

Reply:
I thought internal types should not be under include/openenclave/bits.  If this file has other non-public parts, then maybe we should file a separate issue to track cleaning up this directory.

Reply:
Filed an issue #4132.